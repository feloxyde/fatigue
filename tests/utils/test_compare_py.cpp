#include <iostream>


int main()
{

  std::cout << "some string" << std::endl;
  std::cout << "some newline" << std::endl;
  std::cout << "someemptynewline" << std::endl;
}