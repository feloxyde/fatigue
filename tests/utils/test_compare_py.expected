some string
some newline
someemptynewline
